.buildkite/helm.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs
```

Comment:
In the operator, our .nix files are in the `ci` directory. Can we move these there?

Reply:
yes I missed that